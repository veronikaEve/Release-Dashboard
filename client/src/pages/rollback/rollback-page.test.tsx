import { render } from "@testing-library/react";

import RollbackPage from "./rollback-page";

describe("RollbackPage", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<RollbackPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
