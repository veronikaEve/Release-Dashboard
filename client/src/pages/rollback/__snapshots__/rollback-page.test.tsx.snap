// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RollbackPage should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-md-5"
    style="display: grid; gap: 40px;"
  >
    <button
      class="btn btn-primary"
      type="button"
    >
      Rollback 
    </button>
  </div>
</DocumentFragment>
`;
